webapp/channels/src/components/user_settings/display/user_settings_theme/custom_theme_chooser/custom_theme_chooser.tsx

```diff
@@ -220,24 +220,33 @@ export class CustomThemeChooser extends React.PureComponent<Props, State> {
         this.textareaRef.current?.setSelectionRange(0, this.state.copyTheme.length);
     };
 
-    toggleSidebarStyles = (e: MouseEvent<HTMLDivElement>) => {
+    toggleSidebarStyles = (e: MouseEvent<HTMLButtonElement>) => {
         e.preventDefault();
 
         this.sidebarStylesHeaderRef.current?.classList.toggle('open');
```

Comment:
I didn't realize how much of this is using old jQuery-style patterns of manipulating these manually instead of using proper React. I'm very leery that this code will be unreliable since it could lead to collapsing categories or removing ARIA attributes if React decides to re-render the component at a bad time, but as long as we can make this accessible without worrying about that, I think we can revisit this later to make this properly reactive (https://mattermost.atlassian.net/browse/MM-62569)